yarn-project/acir-simulator/src/client/private_execution.test.ts

```diff
@@ -674,19 +679,19 @@ describe('Private Execution test suite', () => {
 
       const commitment = newCommitments[0];
       const storageSlot = computeSlotForMapping(new Fr(1n), owner.toField(), circuitsWasm);
-      expect(commitment).toEqual(await acirSimulator.computeInnerNoteHash(contractAddress, storageSlot, note.preimage));
+      const innerNoteHash = await acirSimulator.computeInnerNoteHash(contractAddress, storageSlot, note.preimage);
+      expect(commitment).toEqual(innerNoteHash);
 
-      // read request should match commitment
-      const nonce = computeCommitmentNonce(circuitsWasm, txNullifier, 0);
+      // read request should match innerNoteHash for pending notes (there is no nonce, so can't compute "unique" hash)
       const readRequest = execGetThenNullify.callStackItem.publicInputs.readRequests[0];
-      expect(readRequest).toEqual(computeUniqueCommitment(circuitsWasm, nonce, commitment));
+      expect(readRequest).toEqual(innerNoteHash);
 
       const gotNoteValue = execGetThenNullify.callStackItem.publicInputs.returnValues[0].value;
       expect(gotNoteValue).toEqual(amountToTransfer);
 
       const nullifier = execGetThenNullify.callStackItem.publicInputs.newNullifiers[0];
       expect(nullifier).toEqual(
-        await acirSimulator.computeNullifier(contractAddress, nonce, note.storageSlot, note.preimage),
+        await acirSimulator.computeInnerNullifier(contractAddress, Fr.ZERO, note.storageSlot, note.preimage),
```

Comment:
Prefer the pattern of assigning constants to a named `const`, and then passing that named `const` to the function. Although I guess that's less of a readability problem in vscode now, where that nice extension gives annotations about the parameter names of the function that's being called.

Reply:
done